CHANGELOG.md

```diff
@@ -6,6 +6,153 @@
+[Let's Encrypt](https://letsencrypt.org/) integration. However, this means
+that some of the keys used to set up your own ingress has changed.
+
+If you were using config under `ingress` purely to get HTTPS, we recommend
+that you delete your entire config section under `ingress` & instead follow
+the new [docs](<- link -> ) on getting HTTPS set up. It's much easier & a lot
+less error prone than the method recommended on 0.4.
+
+If you were using config under `ingress` for other reasons, you may continue
+to do so. The keys under `ingress` have changed, and are now much more in line
+with how many other projects use `ingress` in the [official charts repo](http://github.com/kubernetes/charts/).
+
+#### Custom images
+
+If you are using a custom built image, make sure that the version of the
+JupyterHub package installed in it is now 0.8. It needs to be version 0.7.2 for
+it to work with v0.4 of the helm chart, and needs to be 0.8 for it to work with
+v0.5 of the helm chart.
+
+#### Admin config incompatibility
+
+If you had used the `admin` config section before, you now need to move it under
+`auth`. So if you had config like:
+
+```yaml
+admin:
+   access: true
+   users:
+      - yuvipanda
+```
+
+it should now be:
+
+```yaml
+auth:
+  admin:
+    access: true
+    users:
+        - yuvipanda
+```
+
+### New Features
+
+#### JupyterHub 0.8
+
+JupyterHub 0.8 is full of new features - see [CHANGELOG](https://github.com/jupyterhub/jupyterhub/blob/master/docs/source/changelog.md#080-2017-10-03)
+for more details. Specific features made to benefit this chart are:
+
+1. No more 'too many redirects' errors at scale!
+2. Lots of performance improvements, we now know we can handle up to 4k active users
+3. Concurrent spawn limits (set via `hub.concurrentSpawnLimit`) can be used to limit the concurrent
+   number of users who can try to launch on the hub at any given time. This can be
+   tuned to avoid crashes when hundreds of users try to launch at the same time. It gives
+   them a friendly error message + asks them to try later, rather than spinning forever.
+4. Active Server limit (set via `hub.activeServerLimit`) can be used to limit the
+   total number of active users that can be using the hub at any given time. This allows
+   admins to control the size of their clusters.
+5. Memory limits & guarantees (set via `singleuser.memory`) can now contain fractional
+   units. So you can say `0.5G` instead of having to use `512M`.
+
+And lots more!
+
+#### Much easier HTTPS
+
+It is our responsibility as software authors to make it very easy for admins to set up
+HTTPS for their users. v0.5 makes this much easier than v0.4. You can find the new
+instructions <here> and they are much simpler!
+
+You can also now use your own HTTPS certificates & keys rather than using Let's Encrypt.
+
+#### Better Authentication Support
+
+The following new authentication providers have been added:
+
+1. GitLab
+2. CILogon
+3. ??? (Globus, but not here yet)
```

Comment:
3. Additional providers may be added in future releases (i.e. Globus)

Reply:
I'm mostly leaving this as a placeholder to see if #227 lands in time, if not I'll just remove it. Mostly here to incentivize @NickolausDS and @rpwagner :)